homeassistant/components/velux/cover.py

```diff
@@ -160,3 +161,4 @@ async def async_set_cover_tilt_position(self, **kwargs):
             await self.node.set_orientation(
                 orientation=orientation, wait_for_completion=False
             )
+        return None
```

Comment:
This shouldn't return anything. We should just remove the if check since the tilt position parameter is required.

Reply:
Should have gotten my first coffee first... ☕ 
Fixed.